otherlibs/win32unix/readlink.c

```diff
@@ -0,0 +1,104 @@
+/***********************************************************************/
+/*                                                                     */
+/*                                OCaml                                */
+/*                                                                     */
+/*            Xavier Leroy, projet Cristal, INRIA Rocquencourt         */
```

Comment:
You need to set the correct author here. Is this mostly copied from Xavier's code, or is it substantially new code?